### ISSUE TITLE
性能达到论文上的水平了吗

### ISSUE THREAD

**Issue by xungeer29** [Tianxiaomo/pytorch-YOLOv4#146]
如题

**Comment by ersheng-ai** [comment]
Systematic performance test on MS COCO dataset for tensorRT model (fp32, fp16 and int8) is under planning.

**Comment by ersheng-ai** [comment]
This is performance on MS COCO validation set (val2017) until now
```
 conf_thresh = 0.001, NMS_IOU_thresh = 0.6, input=416 * 416

 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.46605
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.70384
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.50477
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.26692
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.52448
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.62891
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.34238
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.54948
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.59087
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.38072
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.65378
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.76268
```

**Comment by ersheng-ai** [comment]
## Latest performance data:

- val2017 dataset (416x416)

    | Model type          | AP          | AP50        | AP75        |  APS        | APM         | APL         |
    | ------------------- | ----------: | ----------: | ----------: | ----------: | ----------: | ----------: |
    | Pytorch (TianXiaomo)|       0.466 |       0.704 |       0.505 |       0.267 |       0.524 |       0.629 |
    | ONNX                |    incoming |    incoming |    incoming |    incoming |    incoming |    incoming |
    | TensorRT FP32 + BatchedNMSPlugin | 0.472| 0.708 |       0.511 |       0.273 |       0.530 |       0.637 |
    | TensorRT FP16 + BatchedNMSPlugin | 0.472| 0.708 |       0.511 |       0.273 |       0.530 |       0.636 |

- testdev2017 dataset (416x416)

    | Model type          | AP          | AP50        | AP75        |  APS        | APM         | APL         |
    | ------------------- | ----------: | ----------: | ----------: | ----------: | ----------: | ----------: |
    | DarkNet (YOLOv4 paper)|     0.412 |       0.628 |       0.443 |       0.204 |       0.444 |       0.560 |
    | Pytorch (TianXiaomo)|       0.404 |       0.615 |       0.436 |       0.196 |       0.438 |       0.552 |
    | ONNX                |    incoming |    incoming |    incoming |    incoming |    incoming |    incoming |
    | TensorRT FP32 + BatchedNMSPlugin | 0.412| 0.625 |       0.445 |       0.200 |       0.446 |       0.564 |

**Comment by jicongh** [comment]
Hi，Is there any code to reproduce this mAP result, Thank you

**Comment by GuoQuanhao** [comment]
> ## Latest performance data:
> * val2017 dataset (416x416)
>   
>   
>   
>   Model type
>   AP
>   AP50
>   AP75
>   APS
>   APM
>   APL
>   
>   
>   
>   
>   Pytorch (TianXiaomo)
>   0.466
>   0.704
>   0.505
>   0.267
>   0.524
>   0.629
>   
>   
>   ONNX
>   incoming
>   incoming
>   incoming
>   incoming
>   incoming
>   incoming
>   
>   
>   TensorRT FP32 + BatchedNMSPlugin
>   0.472
>   0.708
>   0.511
>   0.273
>   0.530
>   0.637
>   
>   
>   TensorRT FP16 + BatchedNMSPlugin
>   0.472
>   0.708
>   0.511
>   0.273
>   0.530
>   0.636
> * testdev2017 dataset (416x416)
>   
>   
>   
>   Model type
>   AP
>   AP50
>   AP75
>   APS
>   APM
>   APL
>   
>   
>   
>   
>   DarkNet (YOLOv4 paper)
>   0.412
>   0.628
>   0.443
>   0.204
>   0.444
>   0.560
>   
>   
>   Pytorch (TianXiaomo)
>   0.404
>   0.615
>   0.436
>   0.196
>   0.438
>   0.552
>   
>   
>   ONNX
>   incoming
>   incoming
>   incoming
>   incoming
>   incoming
>   incoming
>   
>   
>   TensorRT FP32 + BatchedNMSPlugin
>   0.412
>   0.625
>   0.445
>   0.200
>   0.446
>   0.564

I want to know the settings you achieved AP, is the default settings in yolov4.cfg?
